LICENSE.md

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2021 Yusuke Tsutsumi
```

Comment:
should this be 2020? I guess it doesn't matter too much.

Also thank you! was going to add a license, but this LGTM.

Reply:
:D. Now that I think about it, it might be more accurately as 2019-2021 YOU.
```suggestion
Copyright (c) 2019-2021 Yusuke Tsutsumi
```